flyctl/app_config.go

```diff
@@ -27,6 +27,7 @@ type AppConfig struct {
 	AppName    string
 	Build      *Build
 	Definition map[string]interface{}
+	Env        map[string]string
```

Comment:
I think there might be benefit to declaring a new type, `Environment`, on which the rest of the functionality of this PR should be implemented on. `Build` works similarly.

Reply:
That sounds good.

Reply:
Actually that seems like it'll add more code for not much benefit — will need to add a `Get`, `Set` and a `Range` (all three are used) methods, or just expose the map again on the `Env` struct. Checking if see if there's any other benefit to pulling this up, or if we can just use one of the stdlib maps like Values.

Reply:
Sorry, misunderstood. I've added a new type called `Environment` which is just `map[string]string`.